workspace-tools/src/generators/library-generator/index.ts

```diff
@@ -40,30 +40,36 @@ export enum LibraryType {
   util = 'util',
 }
 
-export interface DhLibrarySchema {
+export interface LibrarySchema {
   domain: string;
   libraryType: LibraryType;
   name?: string;
+  product: string;
 }
 
-export default async function (tree: Tree, schema: DhLibrarySchema) {
+export default async function (tree: Tree, schema: LibrarySchema) {
   validateParams(schema);
 
   const { fileName: libType, className: libTypeClassName } = names(schema.libraryType);
   const { fileName: libName, className: libClassName } = names(schema.name ?? '');
   const { fileName: libDomain, className: libDomainClassName } = names(schema.domain);
+  const { fileName: libProduct, className: libProductClassName } = names(schema.product);
 
   const libPath = getFinalLibraryPath({
     libDomain,
     libType: schema.libraryType,
     libName,
+    libProduct,
   });
 
   await libraryGenerator(tree, {
     name: getFinalLibraryName(schema.libraryType, libName),
-    directory: `dh/${libDomain}`,
-    tags: `product:dh, domain:${libDomain}, type:${libType}`,
-    prefix: 'dh',
+    directory: `${libProduct}/${libDomain}`,
+    tags: `product:${libProduct}, domain:${libDomain}, type:${libType}`,
+    prefix: `${libProduct}`,
+    importPath: `@energinet-datahub/${libProduct}/${libDomain}/${
+      libName !== '' ? libName : libType
+    }`,
```

Comment:
issue: when I generate a `feature` lib in `dh` with name `test-two`, the import path end up being `@energinet-datahub/dh/admin/test-two`. It should be `@energinet-datahub/dh/admin/feature-test-two`. It's probably the same with other library types.

Reply:
Is this only for feature libs?

Reply:
No. Other lib types use the same pattern too. A list of these libs can be found in the `name` property under the `library-generator`.

![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/abaee8f8-36e8-4fe1-b036-2f082a3c282f)

Reply:
Thanks, I'll fix that 👍